spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/RegionProposalSpec.scala

```diff
@@ -265,10 +265,11 @@ class RegionProposalSpec extends FlatSpec with Matchers {
     layer.evaluate()
     val output = layer.forward(T(T(features), images)).toTable
     val outputExpected = Tensor[Float](
-      T(T(0.0f, 0.0f, 20.999596f, 19.0f),
-      T(0.0f, 0.0f, 12.995603f, 19.0f),
-      T(0.0f, 0.0f, 37.0f, 19.0f),
-      T(0.0f, 0.0f, 29.011127f, 13.003019f)
+      T(T(0.0f, 0.0f, 19.313313f, 19.0f),
```

Comment:
Why you changed the expected output ? Does it means, there's a bug of previous implementation before this commit?

Reply:
yes, original nms implements have been changed in lastest pytorch as it is not exactly same with its meaning. So I also changed nms in bigdl and it's expectation.